infrastructure/quick-lint-js-web/run-apache2.sh

```diff
@@ -9,7 +9,11 @@ set -u
 printf '\n============================================================\n'
 printf '\nrunning web server: http://quick-lint-js.com%s/\n' "${DEV_TLD}"
 printf '\n============================================================\n'
-exec apache2ctl -D FOREGROUND
+
+php-fpm8.1 -c /etc/php/8.1/fpm/php-fpm.conf -F &
+apache2ctl -D FOREGROUND &
+
+wait 
```

Comment:
If we want `wait -n`, we should install Bash in the container and use it for this script. It looks like POSIX sh doesn't support `-n` or an equivalent. =\

Reply:
switched to bash and had to use pids with wait -n